django/db/migrations/state.py

```diff
@@ -680,10 +680,14 @@ def clone(self):
         """Return a clone of this registry."""
         clone = StateApps([], {})
         clone.all_models = copy.deepcopy(self.all_models)
-        clone.app_configs = copy.deepcopy(self.app_configs)
```

Comment:
The `deepcopy` was copying apps and models that were available at `clone.all_models`.